.github/workflows/test-wheel-linux.yml

```diff
@@ -34,86 +34,43 @@ jobs:
     outputs:
       MATRIX: ${{ steps.compute-matrix.outputs.MATRIX }}
     steps:
+      - name: Checkout ${{ github.event.repository.name }}
+        uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8  # v5.0.0
+        with:
+          fetch-depth: 0
```

Comment:
It seems like this and other clones in the matrix step could be `fetch-depth: 1` as they don't need history or tags. I was thinking about doing this with an initial step before anything ran just once too, and then feeding the matrices through, but not sure it is worth the complication.